test/build/50_ch-image.bats

```diff
@@ -319,6 +319,40 @@ EOF
    [[ $output = *"$CH_IMAGE_STORAGE not a builder storage"* ]]
 }
 
+@test 'ch-image rename' {
+   export CH_IMAGE_STORAGE="$BATS_TMPDIR"/rename
+   ch-image pull alpine:3.9
+   run ch-image rename alpine:3.9 alpacino
+   [[ $status -eq 0 ]]
+   echo "$output"
+   [[ $output == *"rename alpine:3.9 to alpacino"* ]]
+   # Image refs not specified.
+   run ch-image rename
+   [[ $status -eq 1 ]]
+   echo "$output"
```

Comment:
Always echo the output before checking anything so errors are visible in the test output.